src/app/search/directives/search-series.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { CATEGORIES, SUBCATEGORIES } from '../../shared/model/story.categories';
+
+@Component({
+  selector: 'publish-search-series',
+  styleUrls: ['search-series.component.css'],
+  template: `
+    <div class="form-group">
+      <!-- TODO: delay search until finish typing -->
+      <input (keyup)="searchByText()" [(ngModel)]="searchText" placeholder="search by title or description"/>
```

Comment:
There are some interesting ways to debounce this in RXJS.  (Searching "angular2 debounce" gives some results from RC5, which still may be applicable).